etc/config.xml

```diff
@@ -50,6 +50,18 @@ xsi:noNamespaceSchemaLocation="urn:magento:module:Magento_Store:etc/config.xsd">
                 <payment_action>authorize_capture</payment_action>
             </omise_offsite_alipay>
 
+            <omise_offline_paynow>
+                <active>0</active>
+                <title>PayNow QR Payment</title>
```

Comment:
It's called 'PayNow QR Payment' here, but just 'PayNow' elsewhere in other labels/titles/comments

Reply:
yes, fixed...